src/Console/Command/FixCommand.php

```diff
@@ -52,6 +52,8 @@
  *
  * @final
  *
+ * @TODO 4.0: mark as final
```

Comment:
But we have to refactor the CheckCommand first, because CheckCommand extends FixCommand.

Reply:
that's one option. other is to at least indicate this by `@final` annotation